src/main/java/org/vaadin/example/bookstore/ui/inventory/InventoryView.java

```diff
@@ -94,39 +95,73 @@ public void showError(String msg) {
         Notification.show(msg);
     }
 
-    public void showSaveNotification(String msg) {
+    /**
+     * shows a message in a notification.
```

Comment:
I think we could be more elaborate what is a "notification". Maybe simply something like `Show a temporary popup notification to the user.` Also in these cases it might be good to have  a link `@see Notification:show`  

wdyt @tmattsso ?

Reply:
Done.